### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tanushyara**
[Mailcot.com](https://mailcot.com)- Mailcot provides email marketing services.
[migomail.com](https://migomail.com)- migomail provides transactional smtp server.

### CODE REVIEW

**Comment by FatGrizzly** [comment]
isn't these both demos?
image

**Comment by FatGrizzly** [comment]
image

File: README.md [hunk 1]
```diff
@@ -650,6 +650,8 @@ Table of Contents
 
 ## Email
 
+  * [Migomail(https://migomail.com/) – migomail provides transactional smtp server.
+  * [Mailcot(https://mailcot.com/) – mailcot provides email marketing services.
```
Comment (ripienaar) [inline README.md:654]: please update with details of the free tier, does not seem to be a free service at all.

**Comment by ripienaar** [comment]
closing stale PR

**Pull request closed without merging**